sway-lib-std/src/context.sw

```diff
@@ -6,11 +6,50 @@ use ::call_frames::contract_id;
 use ::registers::balance;
 
 /// Get the balance of coin `asset_id` for the current contract.
+///
+/// # Arguments
+///
+/// * `asset_id`: [ContractId] - The asset of which the balance should be returned.
+///
+/// # Returns
+///
+/// * [u64] - The amount of the asset which the contract holds.
+///
+/// # Examples
+///
+/// ```sway
+/// use std::{context::this_balance, token::mint, call_frames::contract_id};
+/// 
+/// fn foo() {
+///     mint(50);
+///     assert(this_balance(contract_id()) == 50);
+/// }
+/// ```
 pub fn this_balance(asset_id: ContractId) -> u64 {
     balance_of(asset_id, contract_id())
 }
 
 /// Get the balance of coin `asset_id` for the contract at 'target'.
+///
+/// # Arguments
+///
+/// * `asset_id`: [ContractId] - The asset of which the balance should be returned.
+/// * `target`: [ContractId] - The contract of which the balance should be returned.
+///
+/// # Returns
+///
+/// * [u64] - The amount of the asset which the `target` holds.
+///
+/// # Examples
+///
+/// ```sway
+/// use std::{context::balance_of, token::mint, call_frames::contract_id};
+/// 
+/// fn foo() {
+///     mint(50);
+///     assert(this_balance(contract_id(), contract_id()) == 50);
```

Comment:
this example does not even use balance_of?

Reply:
Whoops, I think this is a copy pasta error